scripts/collector/mysql/collect-data.sh

```diff
@@ -226,8 +226,8 @@ for f in $(ls -1 sql/*.sql | grep -v -e _mysqlcollector.sql -e init.sql -e passw
 do
   fname=$(echo ${f} | cut -d '/' -f 2 | cut -d '.' -f 1)
     ${SQLCMD} --user=$user --password=$pass -h $host -P $port --force --table  ${db} >output/opdb__${fname}__${V_TAG} <<EOF
-SET @DMASOURCEID='${DMA_SOURCE_ID}' ; 
-SET @DMAMANUALID='${V_MANUAL_ID}' ;
+SET @DMA_SOURCE_ID='${DMA_SOURCE_ID}' ; 
+SET @DMA_MANUAL_ID='${V_MANUAL_ID}' ;
```

Comment:
This was a bit of a personal preference.  I find it easier to follow snake case or camel-case.  We can reject this if needed.

Reply:
LGTM